### ISSUE TITLE
Enable Content Filtering completion endpoint

### ISSUE THREAD

**Issue by tomhicks** [Njerschow/openai-api#29]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `openai-api@1.2.1` for the project I'm working on.

In order to use OpenAI in production, we are required to use the content filtering API, which is a special completion engine, as described here [https://beta.openai.com/docs/engines/content-filter](https://beta.openai.com/docs/engines/content-filter).

Also, unrelated but I seemingly needed to patch the export to satisfy TypeScript, which is in the diff. Feel free to consider that separately.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/openai-api/config.js b/node_modules/openai-api/config.js
index 60b4a32..6415103 100644
--- a/node_modules/openai-api/config.js
+++ b/node_modules/openai-api/config.js
@@ -1,5 +1,5 @@
 const DEFAULT_ENGINE = 'davinci';
-const ENGINE_LIST = ['ada', 'babbage', 'curie', 'davinci', 'davinci-instruct-beta', 'curie-instruct-beta'];
+const ENGINE_LIST = ['ada', 'babbage', 'curie', 'davinci', 'davinci-instruct-beta', 'curie-instruct-beta', 'content-filter-alpha-c4'];
 const ORIGIN = 'https://api.openai.com';
 const API_VERSION = 'v1';
 const OPEN_AI_URL = `${ORIGIN}/${API_VERSION}`
diff --git a/node_modules/openai-api/index.d.ts b/node_modules/openai-api/index.d.ts
index c126d0c..9755c4b 100644
--- a/node_modules/openai-api/index.d.ts
+++ b/node_modules/openai-api/index.d.ts
@@ -61,5 +61,5 @@ declare module 'openai-api' {
         encode(str: string): number[];
         search(opts: SearchOpts): Promise<Search>;
     }
-    export = OpenAI;
+    export default OpenAI;
 }
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by Njerschow** [comment]
closed by 5bfcdad6ba18157d89075d6361acda23800f0d0e

**Comment by Njerschow** [comment]
closed by 5bfcdad6ba18157d89075d6361acda23800f0d0e
